chainer_chemistry/models/prediction/regressor.py

```diff
@@ -112,13 +123,19 @@ def __call__(self, *args, **kwargs):
         self.metrics = None
         self.y = self.predictor(*args, **kwargs)
         self.loss = self.lossfun(self.y, t)
-        reporter.report({'loss': self.loss}, self)
+
+        # When the reported data is a numpy array, the loss and metrics values
+        # are scalars. When the reported data is a cupy array, sometimes the
+        # same values become arrays instead. This seems to be a bug inside the
+        # reporter class, which needs to be addressed and fixed. Until then,
+        # the reported values will be converted to numpy arrays.
+        reporter.report({'loss': Variable(cuda.to_cpu(self.loss.data))}, self)
```

Comment:
Do we need to wrap it to Variable?

Reply:
Some tests were failing when I tried to report the value itself. Maybe I did something wrong along the way. I will double check and remove `Variable` if everything works.